go.mod

```diff
@@ -71,7 +71,7 @@ require (
 	go.opentelemetry.io/otel/trace v1.14.0
 	go.uber.org/multierr v1.9.0
 	golang.org/x/exp v0.0.0-20230321023759-10a507213a29
-	google.golang.org/api v0.111.0
+	google.golang.org/api v0.114.0
```

Comment:
Handy link for changelog reference: https://github.com/googleapis/google-api-go-client/releases